With Java 9: Cannot find symbol javax.annotation.Generated

Compiling with `java_grpc_library()` with Java 9 raises the error:
```
cannot find symbol
@javax.annotation.Generated(
^
symbol: class Generated
location: package javax.annotation
1 error

FAILURE: Build failed with an exception.
```

See related gRPC discussion: https://github.com/grpc/grpc-java/issues/3633

The native `java_grpc_library` in Bazel was fixed by adding a dependency on javax.annotation-api: https://github.com/bazelbuild/bazel/pull/5017

--- a/java/BUILD.bazel
+++ b/java/BUILD.bazel
@@ -19,6 +19,7 @@ java_library(
     exports = [
         "@com_google_guava_guava//jar",
         "@com_google_protobuf//:protobuf_java",
+        "@javax_annotation_javax_annotation_api//jar",
     ],
     visibility = ["//visibility:public"],
 )

